components/class-go-newrelic-browser.php

```diff
@@ -83,10 +94,22 @@ public function settings()
 			?>
 		</div>
 		<?php
+
 	}// end settings
 
+	private function enable( $enabled )
+	{
+		return update_option( $this->slug . '-enabled', (bool)$enabled );
+	}//end enable
+
+	private function enabled()
```

Comment:
`enabled` and `enable` are awfully close - one letter? I'd change one to something else, maybe `is_enabled()` to avoid any typo confusion.